civihr_employee_portal/js/reports.js

```diff
@@ -331,13 +330,57 @@
     }
 
     /**
+     * Init angular in reports custom page,
+     * and add the calendar icon
+     *
+     */
+    HRReport.prototype.initAngular = function() {
+        var self = this;
+
+        require([
+            'common/angular',
+            'common/services/angular-date/date-format',
+            'common/directives/angular-date/date-input',
+            'common/moment',
+            'common/filters/angular-date/format-date'
+        ], function() {
+            angular.module('hrr', [
+                'ngAnimate',
+                'ui.bootstrap',
+                'common.angularDate'
+            ]).controller('FiltersController', function($scope, $timeout) {
+                $scope.format = 'dd/MM/yyyy';
+
+                $('[ng-model]').each(function () {
```

Comment:
If what you are trying to do is to set up the scope property that controls the `is-open` property of the datepicker, you can set everything up in the view, please look [here](https://github.com/civicrm/civihr/blob/develop/uk.co.compucorp.civicrm.appraisals/views/dashboard.html#L214-L217) for an example:
```html
<input
    type="text"
    class="form-control"
    id="manager-due-to"
    datepicker-popup
    ng-model="dashboard.filters.cycle_manager_appraisal_due.to"
    ng-init="dashboard.calendars.managerTo = false"
    ng-click="dashboard.calendars.managerTo = !dashboard.calendars.managerTo"
    is-open="dashboard.calendars.managerTo">
```
so basically you would do this in your `generateFilterInputStructure` function